.github/workflows/code_quality.yml

```
@@ -4,30 +4,113 @@ on:
+          # `../mrploch-development/...` imports and the `../ploch-common/...` project
+          # references resolve to paths that do not exist inside the container and the
+          # clones are effectively invisible. Mounting them at /data/<name> makes
+          # /data/project/../<name> resolve exactly as it does on the runner.
+          #
+          # No `--baseline` here on purpose. The previous value was
+          # `--baseline,./qodana.sarif.json`, pointing at a 4 MB SARIF checked in at the
+          # repository root that was produced by QDNET 231 against a *different* solution
+          # (Ploch.Common). A baseline from another project cannot suppress anything
+          # meaningful here and only risks failing the scan, so the run is unbaselined.
+          # Reinstate a baseline only after a green scan of this repository has produced one.
+          args: -v,${{ runner.workspace }}/ploch-common:/data/ploch-common,-v,${{ runner.workspace }}/mrploch-development:/data/mrploch-development
```

actionlint (1.7.12)

[error] property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)